README.md

```diff
@@ -2,117 +2,150 @@
-- Major software development required
-- Too imposing on how teams manage their operations
-
-## How to submit
-Project ideas should be submitted through the [Github issues](https://github.com/ga4gh/TASC/issues) on the TASC repository.
-
-## What should be included in the submission
-- Clear problem statement
-- Explanation of how this impacts alignment between GA4GH standards
-- Background research and landscape analysis
-- Proposed solution
-
-## What to Expect
-
-A project submitted to the TASC will go through several steps.
-- **New issues**: Issues will enter a queue for review. 
-    - Inputs expected from the issue submitter.
-- **Assessment**: After submission, the TASC leadership team will review the issue to ensure that it contains the required elements as stated in the previous section and if it meets the guidelines of what kinds of projects TASC will undertake.  If it needs additional information, the submitter will be notified, the issue will be returned to the “New Issues” queue, and it will be tagged for Revision. 
-    - Inputs expected from the TASC leadership and the issue submitter.
-- **Under Discussion**: Once a project has been accepted by TASC leadership, it is passed to the team for discussion.  As in the assessment stage, the full team can also request more information or determine if the meets the TASC guidelines as they analyze the issue in more detail.  In this stage, the team determines a way forward for the project. 
-    - Inputs expected from the TASC team
-- **Solution Building**:  The team works on a solution to the problem. 
-    - Inputs expected from the TASC team
-- **Blocked**: A temporary holding place while we wait for external issues to be resolved.
-    - Inputs expected from the TASC team
-- **Resolution**: A resolution is proposed and discussed by the team.  The team and leadership vote on the solution.  If work needs to continue, the project is sent back to the Under Discussion or Solution Building phase. 
-    - Inputs expected from the TASC leadership.
-- **Complete**: The agreed-upon solution is disseminated to the GA4GH workstreams.  Note: The Steering Committee has the ability to overrule TASC solutions that they find to be problematic. 
-    - Inputs expected from the TASC leadership and the Steering Committee.
-
-You can track the status of your issue in the [TASC’s Github projects board](https://github.com/orgs/ga4gh/projects/7).
+The TASC Leadership is composed of two volunteer co-leads who are responsible for convening the group, setting its agenda, facilitating decisions, and serving as the central point of communication for stakeholders. The CPO decides who holds the co-lead positions.
+
+#### Chief Product Officer (CPO)
+- **Sasha Siegel** - Chief Product Officer
 
+The CPO ensures that the team creates GA4GH-wide technical alignment and that the needs of GA4GH are appropriately considered in any TASC outputs.
+
+#### Work Stream Representatives
+
+| Member | Work Stream |
```

Comment:
Need to update the list of Members to include: 
- Add: Mónica Muñoz Torres (Clin/Pheno), Benjamin Berk (Clin/Pheno), Jaime Guidry Auvil (DURI), Melissa Cline (GKS), Alex Wagner (GKS), Tom Connor (Data Security), Larry Babb (GKS), Brian O'Connor (GIF), maybe others.

Reply:
Dear @mamanambiya -- I saw the PR was merged, but there are still errors. The list of TASC representatives is not yet complete. Also, my name is misspelled. Please see the comment above.